phpcs.xml.dist

```diff
@@ -13,7 +13,9 @@
 
     <!-- Paths to check -->
     <file>./bin</file>
-    <file>./config</file>
+    <file>./config/autoload</file>
+    <file>./config/pipeline.php</file>
+    <file>./config/routes.php</file>
```

Comment:
Many of the files under the config tree are not namespaced, which was causing a lot of tedium fixing CS. I've restricted to the files we really want to test.